File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:

        freq = {}
        def calfreq(self, n: int, l: List[int]) -> int:
            count = 0;
            for i in nums:
                if n == i:
                    count+=1
            return count;
        
        uniqlist = list(set(nums));
        for j in uniqlist:
            freq[j] = calfreq(self, j, nums)

        sorted_keys = sorted(freq, key=lambda k: freq[k], reverse=True)
        ans =[];
        for i in range(k):
            ans.append(sorted_keys[i]);
        return ans;

        

        
        




        

File: Data Structures & Algorithms/majority-element/submission-0.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        // bruteforce
        for (int val : nums) {
            int freq = 0;
            for (int el : nums) {
                if (val == el) {freq++;}
            }
            if (freq > nums.size()/2) {return val;}
        }

        return -1;

    }
};